deployer/deployment-controller/src/test/k8s/experiment-kustomize/app-base/deployment.yaml

```
@@ -0,0 +1,80 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: app
+  labels:
+    app.kubernetes.io/name: $(NAME)
+    app.kubernetes.io/component: payara-app
+    app.kubernetes.io/part-of: $(PROJECT)-$(STAGE)
+    app.kubernetes.io/managed-by: payara-cloud     
+    app.kubernetes.io/version: $(VERSION)
+  annotations:
+    # These annotations define the deployment
+    # Variables like NAME, PROJECT, STAGE are fetched from these
+    payara.cloud/artifact-url: XXX
+    payara.cloud/artifact-version: XXX
+    payara.cloud/project: XXX
+    payara.cloud/stage: XXX
+    payara.cloud/artifact: XXX
+    payara.cloud/domain: 9ba44192900145a88bfb.westeurope.aksapp.io
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: $(NAME)
+      app.kubernetes.io/part-of: $(PROJECT)-$(STAGE)
+      app.kubernetes.io/component: payara-app
+  template:
+    metadata:
+      name: app
+      labels:
+        app.kubernetes.io/name: $(NAME)
+        app.kubernetes.io/instance: $(NAME)
+        app.kubernetes.io/component: payara-app
+        app.kubernetes.io/part-of: $(PROJECT)-$(STAGE)
+        app.kubernetes.io/managed-by: payara-cloud   
+      annotations:
+        payara.cloud/artifact-url: $(URL)
+    spec:
+      initContainers:
+        - name: download-app
+          # This curl image downloads the artifacts from URL in annotation
+          # into deployments volume
+          image: curlimages/curl:latest
+          command:
+            - sh
+          args: ["-c", "cd /deployments && curl -O `cat /podinfo/url`"]
+          volumeMounts:
+            - mountPath: /deployments
+              name: deployments
+            - mountPath: /podinfo
+              name: podinfo
+      containers:
+      - name: micro
+        image: payara/micro:latest
```

```suggestion
        image: payara/micro:jdk11
```
As we are meant to be doing this on JDK11